src/app/container/dockerfile/dockerfile.component.ts

```diff
@@ -42,11 +42,11 @@ export class DockerfileComponent {
   }
   content: string;
   filePath: string;
-  nullContent: boolean;
   public published$: Observable<boolean>;
   public downloadFilePath: string;
   public customDownloadHREF: SafeUrl;
   public customDownloadPath: string;
+  public loading = false;
```

Comment:
Should initially be set to true;

Loading should be set to false once:
- this.containersService.dockerfile(...) succeeds
- this.containersService.dockerfile(...) fails
- there is no this._selectedVersion
- and possibly more

Not noticeable for this component because the selected version is passed in as Input() and takes no time to get selected version and getting the dockerfile doesn't use multiple ga4gh endpoints.